cloudify_nsx/security/group.py

```diff
@@ -65,6 +65,10 @@ def create(**kwargs):
             raise cfy_exc.NonRecoverableError(
                 "Security group '%s' already exists" % group['name']
             )
+        elif use_existing:
+            raise cfy_exc.NonRecoverableError(
+                "Security group '%s' does not exists" % group['name']
```

Comment:
Does not exist (rather than exists) (there are more of these below, but I won't comment all of them)